app/controllers/nodes_controller.rb

```diff
@@ -18,18 +18,34 @@ def edit
   # POST /nodes
   def create
     @node.label = 'unnamed' unless @node.label.present?
-    if @node.save
-      track_created(@node)
-      flash[:notice] = 'Successfully created node.'
-      redirect_to @node
-    else
-      parent = @node.parent
-      if parent
-        redirect_to parent, alert: @node.errors.full_messages.join('; ')
-      else
-        redirect_to summary_path, alert: @node.errors.full_messages.join('; ')
+    @node.save!
+    track_created(@node)
+    flash[:notice] = 'Successfully created node.'
+    redirect_to @node
+  end
+
+  def multiple
+    if params[:nodes][:parent_id].present?
+      @parent = Node.find(params[:nodes][:parent_id])
+    end
+
+    list = params[:nodes][:list].split(/\n/).map(&:strip).select(&:present?)
```

Comment:
Maybe `#lines` gives us something here (instead of split):

https://github.com/dradis/dradis-ce/blob/add-multiple-nodes/app/controllers/evidence_controller.rb#L57

I've updated `Evidence#create_multiple` to use `:strip` instead of `:chomp`